setuptools_rust/setuptools_ext.py

```diff
@@ -215,6 +215,9 @@ def get_tag(self):
                         self.bdist_dir,
                         "macosx-{}-universal2".format(macos_target)
                     )
+
+                if self.universal:
+                    return "py3", "none", plat
```

Comment:
I think the correct `universal` tag is `py2.py3` ?

Also, this should proabably raise a `ValueError` if the binding is not `Binding.NoBinding`, because PyO3 and rust-cpython both depend on the CPython API so anything built with those bindings is definitely not universal.